modules/serverless-rn-1-37-0.adoc

```diff
@@ -0,0 +1,101 @@
+// Module included in the following assemblies
+//
+// * about/serverless-release-notes.adoc
+
+:_content-type: REFERENCE
+[id="serverless-rn-1-37-0_{context}"]
+= Red Hat {ServerlessProductName} 1.37
+
+{ServerlessProductName} 1.37 is now available. New features, updates, fixed issues, and known issues that pertain to {ServerlessProductName} on {ocp-product-title} are included in the following notes:
+
+[id="new-features-1-37-0_{context}"]
+== New features
+
+[id="new-features-eventing-1-37-0_{context}"]
+=== {ServerlessProductName} Eventing
+
+* {ServerlessProductName} now uses Knative Eventing 1.17.
+
+* {ServerlessProductName} now uses Knative for Apache Kafka 1.17.
+
+* {ServerlessProductName} now supports deployment on ARM architecture as a Generally Available (GA) feature.
```

Comment:
This statement is also present in the 1.36 release notes, so I think we should not have it here any more.